### ISSUE TITLE
[BUG] Nothing Phone 1, white screen after granting permissions

### ISSUE THREAD

**Issue by willthehuman** [moseoridev/WarpShare#21]
**Describe the bug**
After granting permissions, nothing appears on the app. (White screen). When trying to share from the phone, no devices appear. The device does not appear when trying to share from my Mac. I toggled in the settings "Discoverable".

**To Reproduce**
1. Install the APK from the releases
2. Allow the permissions
3. See error

**Expected behavior**
The application should show files. The file sharing should show a list of Apple devices.

**Screenshots**
![Screenshot_20230925-123212](https://github.com/moseoridev/WarpShare/assets/38778465/2024da94-5399-4a82-af01-cde5bdaceba8)

**Smartphone (please complete the following information):**
 - Device: Nothing Phone (1)
 - OS: 2.0.2

**Additional context**
I am connected to the same WiFi with both devices.

**Comment by jasonlessenich** [comment]
Yup, same for me on my Samsung Galaxy Tab S6 Lite (SM-P610) running Android 13 (One UI 5.1)
